package.json

```diff
@@ -86,6 +87,7 @@
     "version": "5.0.0"
   },
   "dependencies": {
+    "deepmerge": "^2.0.1",
```

Comment:
I think we should update the `shallowMerge` function to be a `deepMerge` and remove this dependency

Reply:
When building with `video.js`'s `mergeOptions` it seems like the rollup did not remove the unnecessary code and mpd-parser.js shot up to 680KB. I think it would be better to write a simple deep merge function instead.